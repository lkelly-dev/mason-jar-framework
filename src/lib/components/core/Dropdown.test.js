import React from 'react';
import { shallow } from 'enzyme';
import {
  Dropdown,
  DropdownToggle,
  DropdownFeatherToggle,
  DropdownMenu,
  DropdownItem,
  DropdownCard,
} from "./Dropdown";

describe('Dropdown', () => {
  it('Dropdown matches snapshot', () => {
    const component = shallow(<Dropdown></Dropdown>);
    expect(component).toMatchSnapshot();
  })
});
