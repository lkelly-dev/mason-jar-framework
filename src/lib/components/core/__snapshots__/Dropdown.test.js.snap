// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown Dropdown matches snapshot 1`] = `
<div
  className="dropdown"
/>
`;
